astropy/table/jsviewer.py

```diff
@@ -162,10 +160,13 @@ def write_table_jsviewer(table, filename, table_id=None,
     if table_id is None:
         table_id = 'table{id}'.format(id=id(table))
 
+    table_class = "cell-border compact strip hover order-column" #Nice defaults
```

Comment:
As these classes are options of the DataTables stylesheet and can be combined in different ways, it would be great to have the possibility to specify this in the `show_in_browser` method. http://www.datatables.net/manual/styling/classes